uplink/metainfo/kvmetainfo/objects.go

```diff
@@ -182,9 +182,9 @@ func (db *DB) ListObjects(ctx context.Context, bucket string, options storj.List
 	// case storj.Backward:
 	// 	// backward lists backwards from cursor, including cursor
 	// 	endBefore = keyAfter(options.Cursor)
-	// case storj.Forward:
-	// 	// forward lists forwards from cursor, including cursor
-	// 	startAfter = keyBefore(options.Cursor)
+	case storj.Forward:
+		// forward lists forwards from cursor, including cursor
+		startAfter = keyBefore(options.Cursor)
```

Comment:
This should probably happen now on the satellite side rather than on client side.

Reply:
We don't have direction in Metainfo protobuf.

Reply:
Ah, right... I'm wondering whether we should have it. However I just remembered that @zeebo already built it into the path component encoding and we'll remove reverse iteration, so it might be easier to not have it.

Reply:
Regarding reverse iteration, I added this topic for next Arch Review meeting.

Reply:
I'm out of date on this. I think I've missed some arch reviews- did we talk about reverse iteration? I still have https://review.dev.storj.io/c/storj/storj/+/87 pending to remove the reverse iteration but I don't want it to conflict with what you're doing.

Reply:
@thepaul I moved it to WIP state because amount of work was larger than I was thinking at the beginning. Fell free to merge your changes before this PR. I will take a look at your PR.